pytools/tag.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import annotations
```

Comment:
This should be wired into the documentation, i.e.

- Get a module docstring explaining what it's for and including `.. auto*` directives for the parts being exposed.
- Have `.. automodule` for that included in the package docs.

Reply:
Done, but now the Documentation test is unhappy about something.

Reply:
I'm still not quite certain why the documentation generation fails. I have Sphinx v3.2.1 and 'make html' executes without failing.

Reply:
You have to run it in "nitpicky mode". See the CI config.